src/style.css

```diff
@@ -72,7 +68,23 @@ body {
   background: url(/images/star.svg) center no-repeat;
 }
 
-.product-stars__star:nth-child(-n+4) {
+.product-stars_one .product-stars__star:nth-child(-n+1) {
+  background-image: url(/images/star-active.svg);
+}
+
+.product-stars_two .product-stars__star:nth-child(-n+2) {
+  background-image: url(/images/star-active.svg);
+}
+
+.product-stars_three .product-stars__star:nth-child(-n+3) {
+  background-image: url(/images/star-active.svg);
+}
+
+.product-stars_four .product-stars__star:nth-child(-n+4) {
+  background-image: url(/images/star-active.svg);
+}
+
+.product-stars_five .product-stars__star:nth-child(-n+5) {
   background-image: url(/images/star-active.svg);
 }
```

Comment:
this is the same rule, write it using coma separator

Reply:
done